Frontend/android-native/mobility-common/src/main/res/layout/referral_code.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="360dp"
+    android:layout_height="440dp"
+    android:orientation="vertical"
+    android:gravity="center"
+    android:id="@+id/referral_code"
+    android:background="@drawable/referral_code_background">
+    <LinearLayout
+        android:id="@+id/referral_code_share_layout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:paddingRight="40dp"
+        android:paddingLeft="40dp"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/referral_logo"
+            android:layout_width="150dp"
+            android:layout_height="60dp"
+            android:layout_marginTop="10dp"/>
+
+        <ImageView
+            android:id="@+id/code_image"
+            android:layout_width="288dp"
+            android:layout_height="288dp"
+            android:layout_marginTop="15dp" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="50dp"
+            android:gravity="center"
+            android:layout_marginTop="5dp"
+            android:layout_marginBottom="15dp"
+            android:paddingBottom="5dp">
+            <TextView
+                android:layout_width="0dp"
+                android:layout_weight="1"
+                android:layout_height="match_parent"
+                android:text="Referral\nCode"
```

Comment:
Is it hardcoded
If yes then add translations for this

Reply:
we want in english only